src/Type/Definition/EnumType.php

```diff
@@ -71,12 +71,10 @@ public function getValue($name)
         return $lookup[$name] ?? null;
     }
 
-    /**
-     * @return ArrayObject<string, EnumValueDefinition>
```

Comment:
can keep?

Reply:
Unfortunately, not--code sniffer complains about it being "unecessary". Sigh.